From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <rvolz@mit.edu>
Date: Fri, 2 Aug 2024 16:36:15 -0400
Subject: [PATCH] cmake: c: Clean up install so DLL actually installs on
 Windows

---
 c/CMakeLists.txt | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/c/CMakeLists.txt b/c/CMakeLists.txt
index bfd7ab2..fea8192 100644
--- a/c/CMakeLists.txt
+++ b/c/CMakeLists.txt
@@ -93,8 +93,6 @@ target_include_directories(digital_rf PUBLIC
 )
 target_link_libraries(digital_rf PUBLIC ${HDF5_LIB_TARGETS} PRIVATE ${MATH_LIB})
 set_target_properties(digital_rf PROPERTIES
-    ARCHIVE_OUTPUT_DIRECTORY lib
-    LIBRARY_OUTPUT_DIRECTORY lib
     VERSION ${libdigital_rf_VERSION}
     SOVERSION ${libdigital_rf_VERSION_MAJOR}
     INTERFACE_libdigital_rf_MAJOR_VERSION ${libdigital_rf_VERSION_MAJOR}
@@ -105,10 +103,10 @@ set_target_properties(digital_rf PROPERTIES
     POSITION_INDEPENDENT_CODE ON
 )
 
-install(TARGETS digital_rf EXPORT libdigital_rfTargets
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+install(TARGETS digital_rf
+    EXPORT libdigital_rfTargets
 )
+
 install(FILES
     ${CMAKE_CURRENT_BINARY_DIR}/include/digital_rf.h
     ${CMAKE_CURRENT_BINARY_DIR}/include/digital_rf_version.h
-- 
2.46.0

